.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Why are we adding this?

Reply:
This was added either by `lint-staged` or by `husky`. This directory and files inside are generated by the `prepare` script (which is `npx husky install`). Because those files are generated, it probably makes sense that `husky` (or `lint-staged`) wants them to be ignored.

> how to run the linter without having to commit

Nothing has changed :-) `npm run phplint` or `bin/php-lint.sh`.

The only change there is that `php -l` was replaced by `phplint` which is a wrapper around `php -l`. The difference is that `phplint` accepts multiple filenames as command-line arguments.